client/state/data-layer/wpcom/sites/users/test/index.js

```diff
@@ -12,6 +12,8 @@ import {
 	receivePostRevisionAuthorsSuccess,
 } from '../';
 
+const times = ( n, cb ) => [ ...new Array( n ).keys() ].map( ( k ) => cb?.( k ) ?? k );
```

Comment:
I think we should be able to simplify this by reusing the `times()` output below.

```suggestion
const times = ( n ) => Array.from( { length: n } ).map( ( _, k ) => k );
```

Then we can change:

```
const users = times( nbUsers, ( id ) => ( { id } ) );
```

to:


```
const users = ids.map( ( id ) => ( { id } ) );
```

in both instances, because `ids` will already contain an array of numbers.

Reply:
Fixed with https://github.com/Automattic/wp-calypso/pull/82642/commits/1b32b2d6326f3a49ce04ad842792a2b7a77727e9